client/src/components/RatingsAndReviews/ReviewsList.jsx

```diff
@@ -26,6 +24,7 @@ const ReviewsList = () => {
       .then((res) => {
         let reviews = res.data.results;
         setReviews(reviews);
+        console.log(reviews);
```

Comment:
Not sure if this was intentional but wanted to flag that it was left here just in case!